recipes/llvmlite/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "llvmlite" %}
+{% set version = "0.20.0" %}
+{% set sha256 = "b2f174848df16bb9195a07fec102110a06d018da736bd9b3570a54d44c797c29" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+    
+build:
+  skip: True  # [win and py2k]
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - enum34  # [py2k]
+    - llvmdev
```

Comment:
This probably needs to be pinned to 5.0.0 or similar. Though would like @isuruf to advise here as he is more closely involved with this change.

Reply:
Yes, for `0.20.0` it should be exactly `5.0.0`

Reply:
See https://github.com/numba/llvmlite#compatibility